R/gs_b.R

```diff
@@ -19,13 +19,13 @@
 #' Default boundary generation
 #'
 #' `gs_b()` is the simplest version of a function to be used with the
-#' `upper` and `lower` arguments in `gs_prob()`,
-#' `gs_power_nph()` and `gs_design_nph()`;
-#' it simply returns the vector input in the input vector `Z` or,
-#' if `k` is specified, `par[k]j` is returned.
+#' `upper` and `lower` arguments in `gs_prob_combo()`,
```

Comment:
In additional to `gs_prob_combo`, `gs_power_npe` and `gs_design_npe`, shall we add `pmvnorm_combo()`? https://github.com/Merck/gsDesign2/blob/main/R/utility_combo.R#L146

Reply:
I added `pmvnorm_combo()` in https://github.com/Merck/gsDesign2/pull/415/commits/59d6b6fc6ce6c543df2395777df6d7fe9b651f7b

Also, I noticed that the function arguments are different for `gs_prob_combo()` and  `pmvnorm_combo()`. These functions use `lower_bound` and `upper_bound`, so I updated that too